service/src/main/java/de/adorsys/psd2/sandbox/xs2a/service/ais/AccountSpiImpl.java

```diff
@@ -104,6 +104,7 @@ public AccountSpiImpl(TestDataService testDataService, TestDataMapper testDataMa
     }
```

Comment:
Please remove logfile above and gitignore `service/logs`

Reply:
Done